rpa-home/modules/ROOT/pages/anypoint-migration-overview.adoc

```diff
@@ -0,0 +1,41 @@
+= Anypoint Platform Migration Overview
+
+Starting July 1st, RPA Manager integrates with Anypoint Platform to unify the login experience and provide support for external identities (IDP).
```

Comment:
```suggestion
Starting July 1st, RPA Manager integrates with Anypoint Platform to unify the login experience and provide support for external identities (IdP).
```

Reply:
(To be consistent with AIM: https://docs.mulesoft.com/access-management/external-identity-index)